\documentclass{report}

\usepackage{graphicx}
	\graphicspath{{./images/APBIO}}
\usepackage[T1]{fontenc}
\usepackage{bold-extra}
\usepackage{lmodern}
\usepackage{hyperref}
\usepackage{array}
\input{preamble}
\input{macros}
\input{letterfonts}

\title{\Huge{AP Biology}\\ Notes}
\author{Me. I am Him.}
\date{11/28/2022}

\begin{document}

\maketitle
\newpage
\pdfbookmark[section]{\contentsname}{toc}
\tableofcontents
\pagebreak

\chapter{Chemistry of Life}
\section{Scientific Method}
\section{Feedback}
\nt{Positive and Negative feedback. Need to know.}
\section{Macromolecules}


\newpage
\chapter{Cell Structure and Function}
\nt{\textbf{All} cells have the following;
	\begin{enumerate}
		\item Surrounding Membrane
		\item Chromosomes
		\item Cytosol
		\item Ribosomes.
	\end{enumerate}}
\dfn{Prokaryotic Cells}{They are defined as being, having, or including:
	\begin{enumerate}
		\item Non-membrane-bound organelles
		\item Nucleoid
	\end{enumerate}
	\nt{Prokaryotes include \textit{Bacteria} and \textit{Archae}. They both differ genetically.}}
\dfn{Eukaryotic Cells}{They are defined as being, having, or including:
	\begin{enumerate}
		\item Membrane-bound organelles, including a nucleus.
		\item Multicellular potential.
	\end{enumerate}
	\nt{There is a limit on how big a cell can get. This is defined by the square-cube law. As surface area squares, the volume gets cubed.}}
\thm{Cellular Evolution}{Current Evidence indicates that eukaryotes evolved from prokaryotes between 1 and 1.5 billion years ago.
	\nt{There are two current theories on howthis happened:
		\begin{enumerate}
			\item Folding Theory: In short, we believe that parts of the cell membrane folded in on itself, pinched off, and became specialized cells.
			\item Endosymbiotic Theory: The \textit{mitochondria} and \textit{chloroplasts} were once prokaryotic and somehow and for some reason became incorporated into another cell.
\end{enumerate}}}

\subsection{Organelles}
\dfn{Organelle}{Insert Def here.}
\dfn{Nucleus}{Manager of cell functions. It also contains DNA and the nucleolus.}




\chapter{DNA}
\section{DNA and Replication}

\dfn{DNA}{Hereditary information that is found inall cells of the body.}
\nt{DNA Contains the information to make proteins and directs the development of biochemical, atomical, physiological, and --to some extent-- behavioral traits.}

\subsection{Frederick Griffith}

\begin{center}
	Frederick Griffith worked with two strains of a bacterium, a pathogen "S" strain and a harmless "R" strain. He investigated what happened when they were each injected into a mouse.
	\begin{tabular}{rl}
		& $\bullet$ Living 'S' cells = dead mouse \\
		& $\bullet$ Living 'R' cells = alive mouse \\
		& $\bullet$ Heat killed 'S' cells = alive mouse \\
		& $\bullet$ Living 'R' cells + heat killed 'S' cells = dead mouse. \\
	\end{tabular}
	\nt{Note how the mouse is alive with heat killed 'S' cells but dead when those cells are added to 'R' cells. In conclusion, the heat killed 'S' cells are able to infect and take over the living 'R' cells and kill the mouse.}
\end{center}

\subsection{Structure}
\dfn{Double Helix}{Specifically, DNA is two anti-parallel sugar-phosphate backbones, with the ntriogenous bases paired in the molecules interior.}
\nt{All nitrogen bases have a specific partener. For example, Thymine (T) only goes with Adenine (A), and Guanine (G) only goes with Cytosine (C).}

\subsection{Packaging and Organization}
\dfn{Chromatin}{DNA and all associated proteins.}
\dfn{Heterochromatin}{Highly Compacted DNA}
\dfn{Euchromatin}{DNA that is able to be transcribed.}
\nt{If there is no DNA, or the DNA cannot be transcribed, no proteins will be made}
\qs{How is DNA packaged?}{
	\begin{enumerate}
		\item The DNA is wrapped around a \textbf{histone} twice.
	\item Eight histones are then gathered, creating a line. A sort-of beads-on-a-string. These are called \textbf{Nucleosomes}.
	\item These nucleosomes eventually clump up, creating many loop shapes, called \textbf{looped domains}.
	\item These looped domains eventually organize into \textbf{Metaphase chromosomes}.
	\end{enumerate}
\dfn{Histone}{A group of basic proteins found in chromatin. Their tails can be modified.}
\dfn{Nucleosome}{A structural unit of a eukaryotic chromosome, consisting of a determined length of DNA coiled around a core of histones.}
\dfn{Looped Domain}{A basic structural unit of eukaryotic chromatin associated with DNA replication, gene expression, and higher order packaging.}
\dfn{Metaphase Chromosome}{Simply a chromosome in the stage of \textit{metaphase} of the cell cycle.}}

\newpage
\section{DNA Replication}
\dfn{DNA Replication}{The process by which DNA is replicated. It follows a \textit{semi-conservative model}.}
\dfn{Semi-conservative model}{A process where there is one original and one new strand.}
\nt{Copying DNA is easy due to the hydrogen bonds holding the bases together, but also hard thanks to DNA being antiparallel.}
\ex{DNA Replication}{
	\begin{enumerate}
		\item Helicase untwists the DNA and seperates the strands. This is only done in sections. It starts at specific DNA sequences.
		\item Topoisomerase relieves the pressure from the bubble. The bubble forms thanks to helicase working in sections.
		\item Primase then deposits primers from the middle of the buble towards the 5` end.
		\item DNA polymerase III (DP3) adds DNA nucleotides to the 3` end of the primer. The hydrogen bond then forms between the new strand and old strand.
		\item DNA polymerase I (DP1) starts to remove the primer and replaces it with DNA nucleotides. This occurs from 3` to 5`
	\end{enumerate}
	\nt{If the template strand starts from the 3` to 5` end, the new strand will start from the 5` to 3`.}}

\dfn{Leading Strand}{The strand who leads from the half-way-point in the bubble.}
\dfn{Lagging Strand}{The strand who are made before the half-way-point and eventually catch up to the leading strand.
\nt{Individual parts to the lagging strand are called okizaki fragments. They are placed randomly.}}
\dfn{Primase}{Synthesizes RNA primers, using parenta DNA as a template.}
\dfn{Topoisomerase}{Breaks, swivels, and rejoins the parenta DNA ahead of the replication fork, relieving the strain caused by unwinding.}
\dfn{Helicase}{Unwinds and seperates the parental DNA strands.}
\dfn{Single-Strand Binding}{satblize the unwound parental strands.}



\newpage
\section{Genetic Engineering}
\ex{How to Genetic Engineer}{
	\begin{enumerate}
		\item Take a gene and put it into a bacteria
		\item The bacteria will then express the gene and produce the protein.
		\item Extract the protein.
	\end{enumerate}}
\dfn{Transgeneic Organism}{An organism containing DNA from two differing sources.}
\dfn{Recombinant DNA}{A mixed genome derived from two differing sources.}
\dfn{Nucleic Acid Hybridization}{The property where one strand of DNA is complimentary, and therefore will connect to another strand.
	\nt{
		\begin{enumerate}
			\item If you heat up DNA, the strands separate.
			\item If you add a strand that is complimentary and is radioactive, you can find that section. This is used in the \textbf{chain termination method}
		\end{enumerate}}}


\subsection{Cloning}
\dfn{Polymerase Chain Reaction (PCR)}{A process by which sections of a DNA sample is amplified. The process is done in a thermocycler.}
\begin{figure}
	\includegraphics[scale=0.4]{Chapter 13 - PCR.png} \label{PCR}
\end{figure}
\dfn{Restriction Enzyme}{Cleaves DNA sequences as sequence-specific sites, producing DNA fragments with a known sequence at each end.}

\subsection{Sequencing the Genone}
\dfn{Chain Termination Method}{The process by which we add complimentary DNA nucleotides to a known section of DNA. We use a \textbf{thermocycler} to automate this process.
	\nt{Whether the thermocycler chooses a marked nucleotide or not is random. However, when it chooses a marked nucleotide, the process stops.}}
\nt{To analyze the DNA, we use \textit{Gel Electrophoresis}. This sorts them by size and charge.}
\dfn{Restriction Fragment Length Polymorphisms (RFLP's)}{A variation in the length of restriction fragments produced by a given restriction enzyme in a sample of DNA. 
	\nt{Different people have different fragment lengths.}}
	\dfn{Dideoxyidenucleotide}{}


\newpage
\section{Gel Electrophoresis}
\dfn{Gel Electrophorisis}{The process by which you measure and sort strands of DNA, proteins, or other molecules.} 
\qs{How does it work?}{
	$\bullet$ A gel-like filter with many small holes in it sorts the DNA strands. \\
	$\bullet$ First, we place some DNA samples into one end of the filter. \\
	$\bullet$ Second, we run electricity through it, creating a positive and negative end which pushes the DNA strands through the filter. This is where the 'electrophoresis' comes from. \\
	$\bullet$ We can tell which are longer and which are shorter by looking at how far each strand traveled. Shorter strands will travel farther over time. \\
	$\bullet$ We can then stain the sorted groups of DNA, allowing the naked eye to see them. We don't see individual strands, though, we see the groups of them.
	\nt{Fragments are sorted by \textbf{size} and \textbf{charge}. Smaller molecules go farther.}}
\begin{figure}
	\includegraphics[scale=0.3]{Chapter 13 - Gel Electrophoresis.png}
	\label{Gel Electrophoresis}
\end{figure}

\pagebreak
\section{Editing Genes}
\dfn{CRISPR-Cas9}{Clustered Regularly Interspaced Short Palindromic Repeats. This protein has the ability to llcate, cut, and repair or disable a specific \textit{gene}.
	\nt{It was first discovered in bacteriophages as protection against viruses.}
	\nt{There are two parts:
		\begin{enumerate}
			\item Cas9 Protein: Nuclease enzyme that cuts double stranded DNA wherever it is brought.
			\item Guide RNA: Complimentary piece of RNA that binds to a target gene.
		\end{enumerate}}
\nt{If you know the sequence of the gene, you can create the guide RNA.}}
\begin{figure}
	\includegraphics[scale=1]{Chapter 13 - Cas9 RNA.jpg} 
	\label{Cas9}
\end{figure}


\newpage
\section{Gene Expression}

\nt{Protein Synthesis requires two processes:
	DNA $\rightarrow$ RNA $\rightarrow$ Protein}
\dfn{Transcription}{Makes RNA.}
\dfn{Translation}{Makes Proteins.}
\nt{\textit{Transcription} takes place in the nucleus whereas \textit{Translation} takes place at ribosomes in the cytosol. Proteins are modified in the \textit{golgi body}.}

\subsection{The Genetic Code}
\dfn{Codon}{A set of 3 nitrogen bases in DNA that make an amino acid.}
\nt{Only one side of the DNA strand will be used as a template.}
\nt{AUG is the \textbf{ONLY} start codon. This is a must know.}
\nt{There are three stop codons. Memorize them. \textbf{UAA}, \textbf{UGA}, and \textbf{UAG}.}

\subsection{Transcription: Making RNA}
\nt{There are several different types of RNA, all with different shapes and functions. They are all made in the nucleus, but different RNA functions are made in different places. \textbf{Uracil} replaces \textbf{Thymine}.}
\dfn{Gene Unit}{The portion of DNA involved in transcription.}
\dfn{RNA Polymerase}{Like DNA polymerase, RNA polymerase makes RNA.}
\dfn{Promoter}{The part of DNA where RNA polymerase builds. Their purpose is to line up RNA polymerase's active site with the first base to be transcribed.
	\nt{It is 'upstream' from the start of the gene.}}
\dfn{Transcription factor}{HELLLLLLLLPPPPPPPPPPPPPPPPP}
\nt{Unlike how DNA can be copied off other DNA, RNA can \textbf{ONLY} be transcribed from RNA.}

\ex{Three Steps to Transcription:}{
\begin{enumerate}
	\item Initiation
	\item Elongation
	\item Termination
\end{enumerate}}
\dfn{Initiation}{Here, the transcription factors bind to the DNA, RNA polymerase binds to the right spot, and the RNA polymerase begins to unwind the DNA.}
\dfn{Elongation}{Here, RNA polymerase moves along the DNA, untwisting 10 to 20 bases at a time. The complimentary bases are added, and the DNA zips back up behind it. This happens at a rate of 60 nucleotides a second in eukaryotes.}
\dfn{Termination}{In \textit{prokaryotes}, it ends at the terminator. However, in \textit{eukaryotes}, it breaks off after a specific sequence.}

\newpage
\section{RNA Processing}
"""GO BACK TO THE FIRST SLIDES AND RECOPY THE SLIDES.
\nt{All RNA's are virtually the same, but \textit{mRNA} has the instructions to put the amino acids together in the correct order.}
\nt{Pre-\textit{mRNA} must be modified before it's sent to the cytoplasm.}


\subsection{Alteration of mRNA Ends}
\nt{The 5' end gets a Mg Cap. This helps it get out of the nucleus, protects it from degredation, and helps it bind to a ribosome.}
\nt{The 3' end gets a Poly-A tail. The longer the tail, the longer it lasts. It also acilitates the export of \textit{mRNA} from the nucleus and protects it from degredation.}
\dfn{Untranslated Regions}{The region between the mG cap and the coding sequence and between the stop codon and the poly-A tail.}

\subsection{RNA Splicing}
\dfn{RNA Splicing}{The process by which spliceosomes remove introns and push together extrons to create the final sequence.}
\dfn{Spliceosomes}{A conglomerate of various proteins and several small nuclear ribonucleoproteins(\textit{snRNP}s), or \textit{snurps} that recognize the splice sites.}
\dfn{Introns}{A non-coding sequence. They get spliced out and removed.}
\dfn{Extrons}{A coding sequence.They get spliced together after the introns are removed.}
\nt{Must remove introns, they are called spliceosomes. Think of RNA as a movie. Must remove deleted scenes (introns), push together the rest. The final directors cut is made up of the pre-roll (5' cap), the movie itself (extrons), and the credits (3' cap or Poly-A tail).}
\nt{Introns are important for one main reason. They lead to \textit{alternative RNA splicing.}}
\dfn{Alternative RNA Splicing}{The process by which different areas lead to different results from RNA splicing. This works thanks to some areas splicing out some exons.
	\nt{This is how we get hundreds of thousands of proteins from only 21,000 genes.}}

\subsection{Translation}\label{Translation}
\dfn{Translation}{The process to convert the RNA code into a protein.
	\nt{It requires three different types of RNA:
	\begin{enumerate}
		\item mRNA
		\item tRNA
		\item rRNA
	\end{enumerate}}
	It also has three stages:
	\begin{enumerate}
		\item Initiation
		\item Elongation
		\item Termination
	\end{enumerate}}

\dfn{tRNA}{A single molecule consisting of a single RNA strand that is only about 80 nucleotides long.
\nt{It caries an animo acid on the 3` end. Loop two also contains a anti-codon, whcih means that it is complimentary to the codon it's using.}}
\dfn{Wobble}{The relaxing of base pairing on the third base of a(n) (anti)codon. In short, it explains why the third base in a codon can sometimes change without changing the overall protein.}

\subsection{Ribosomes}
\dfn{Ribosome}{Made up of two ribosomal units (large and small) are made up of proteins and \textit{rRNA}
	\nt{A ribosome has three binding sites for tRNA. These are in order:
	\begin{enumerate}
		\item A site: holds the tRNA that carries the next amino acid to be added to the chain.
		\item P site: holds the tRNA that carries the growing polypeptide chain.
		\item E site: is the exit site, where discharged tRNAs leave the ribosome.
	\end{enumerate}}}
\dfn{Initiation}{
	\begin{enumerate}
		\item Small ribosomal subunits binds to \textit{mRNA} (at the start of the codon AUG)
		\item tRNA (carrying methonine) binds to \textit{mRNA} in the P site.
		\item Large ribosomal subunit comes in and arranges itself to the tRNA in the P site.
	\end{enumerate}}
\dfn{Elongation}{
	\begin{enumerate}
		\item Second \textit{tRNA} entes the A-site; codon binds to the anti-codon.
		\item The amino acid on the \textit{tRNA} in the P site 'jumps back' and is bonded to the amino acid on the \textit{tRNA} in the A site.
		\item Translocation - the \textit{tRNA} in the P-site gets moved to the E-site where it gets kicked out while the \textit{tRNA} in the A-site gets moved to the P-site.
		\item Ribosomes move along \textit{mRNA} from 5` to 3`
		\item Repeat.
	\end{enumerate}}
\dfn{Release Factor}{When the ribosome encounters a stop codon, a release factor enters and binds a water molecule to the ribosome. This causes the polypeptide chain to break off from the strand and the ribosome releases.}


\subsection{Point Mutations}
\nt{There are two types of point mutation:
\begin{enumerate}
	\item Base-pair substitutions
	\item Base-pair insertions or deletions.
\end{enumerate}}

\dfn{Substitutions}{The replacement of one nucleotide and its partner with another pair of nucleotides.}
\dfn{Missense Mutation}{A mutation where it still codes for an amin acid, but it may not nessecarily be the correct one.}
\dfn{Nonsense Mutation}{A mutation where an amino acid codon is changed into a stop codon. This leads to a nonfunctional protein.}
\dfn{Insertions and Deletions}{The addition or removal of a nucleotide pair in a gene. This usually leads to massive issues in the reading and translation of the RNA. This is also called a frameshift mutation.}
\dfn{Mutagen}{Physical or chemical agents that can cause mutations. Examples include smoking, radioactivity, and more.}

\dfn{Lac Operon}{When would enzymes \textit{NOT} be made? -> 
\begin{enumerate}
	\item [-] When Lactose is NOT present.
	\item [-] When Glucose IS present.
\end{enumerate}}

\nt{Allolactose is another way to say lactose.}
\nt{The \textit{Lac} Operon and \textit{Trp} Operon are different in that the Lac prosessor (inhibitor) is made active, whereas the Trp processor (inhibitor) is made inactive.}
\dfn{Repressible Operon}{
	\begin{enumerate}
		\item [-] Usually Functions in anabolic pathways.
		\item [-] Synthesizing end products
		\item [-] When end product is present cell allocates to other uses.
		\item [-] Trp Operon.
		\item [-] Genes are active.
		\item [-] Regulatory protein (inhibitor) is made in the inactive form.
		\item [-] The things being made activates the regulatory protein when there is enough of that thing.
	\end{enumerate}}

\dfn{Inducible Operon}{
	\begin{enumerate}
		\item [-] Usually functions in catabolic pathways, digesting nutrients to simpler proteins.
		\item [-] Produce enzymes only when nutrient is available.
		\item [-] Cell avoids making proteins that have nothing to do.
		\item [-] Lac Operon.
		\item [-] Genes are inactive.
		\item [-] Regulatory Protein (inhibitor) is made in active form.
		\item [-] The thing being broken down inactivates the regulatory protein when that thing is around
	\end{enumerate}}

\nt{This \textit{ONLY} in Prokaryotes. Control in Eukaryotes is completely different thanks to their DNA being different.}

\ex{Control in Eukaryotes}{
	\begin{enumerate}
		\item DNA gets transcribed into pre-mRNA
		\item pre-mRNA gets spliced to the proper mRNA
		\item mRNA gets translated into proteins.
		\item The proteins then do their thing.
	\end{enumerate}
	We control this by stopping translation and transcription. It's hard to explain, but think of it as a main water line to your home.
	We stop the main line, then go throughout the house draining the individual sections by turning on the sink.}

\newpage
\section{Regulating DNA Structure}

\dfn{Histone Acetylation}{
	\begin{enumerate}
		\item [-] Acetylated histones seperate from each other; DNA is looser.
		\item [-] Why? Positive Charges are neutralized; no charge to attract.
		\item [-] Allows for transcription
	\end{enumerate}
}
\dfn{Methylation}{
	\begin{enumerate}
		\item [-] Adding methyl groups to DNA to prevent transcription.
		\item [-] Methyl groups are also added to histone proteins.
	\end{enumerate}
}











\newpage
\begin{figure}
	\centering
	\Huge{Thanks for reading}
	\includegraphics[scale=0.2]{dabloon.jpg}
	\label{dabloonia}
\end{figure}
\end{document}